notifications.py

```diff
@@ -53,10 +54,15 @@ def parse_cb_defense_notifications_get_incidentids(response):
     return incidentids
 
 def psc_threathunter_check(response):
-    if 'notifications' in response:
+
+    if response['notifications'] == []:
+        logger.info("There are currently no notification responses to be pulled.")
         return None
 
-    return True
+    if 'threatHunterInfo' in response['notifications'][0]:
```

Comment:
This only checks the first element what if the first element is not a threatHunterInfo? Do we need to check if threatHunter is present especially since the threat hunter parsing one does all three? It also helps to reduce your complexity in `def parse_notifications()`

Reply:
We in theory don't need to check if that is present as I don't believe we do any additional processing beyond parsing the threat_hunter type

Reply:
We need to determine whether the response is coming from Threat Hunter or the PSC cause they are different JSON formats. Notification PSC JSON has "ThreatInfo" as a unique key and ThreatHunter JSON format has "ThreatHunterInfo" which is what i am checking for.  As we have determined, there are 3 types that can be in the data: "Active_Threat", "Policy_Action" and "Threat_Hunter".  Determining whether the Threat_Hunter Type is in the data is not safe cause we can very well pull ThreatHunter data that has just "Active_Threat" and "Policy_Action".

Reply:
Removed function. Updates have been pushed